product_docs/docs/biganimal/release/getting_started/managing_cluster.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Managing a cluster"
+description: Describes options for managing clusters
+---
+
+## Pausing and resuming clusters
+
+Pausing a cluster allows you to save on compute costs without losing data or cluster configuration settings. There is no additional cost for paused clusters, although you must still pay for any disk and object storage that the paused cluster requires. The ability to pause clusters gives you the flexibility to configure applications that don't separate storage and computing. You can use the BigAnimal API or CLI to pause and resume clusters, which allows you to schedule pauses.
+
+Pausing a cluster drops all existing actions on the cluster, blocks future actions such as editing the cluster, and stops monitoring. While paused, you cannot make or maintain connections, or read, write, or edit cluster settings. Automatic backups are paused. However, backups taken while the cluster was running remain until their retention period expires. While paused, clusters are not upgraded or patched, but upgrades are applied as part of the reactivation process.
```

Comment:
We should also add and warn user for below.
We are keeping the cluster in paused state for maximum 7 days and if user defined the backup retention period for that cluster which is less than 7 days, it might possible that WALs may get deleted from backup object store and resumption might fail.

but @amrita-suresh have better wording for this.

Reply:
Added a Note about write-ahead logs (WALs). I think this is sufficient, but if not let me know what you think